internal/serverinstall/ecs.go

```diff
@@ -1886,6 +1887,7 @@ const odrRolePolicy = `{
         "ecs:ListTasks",
         "ecs:DescribeTasks",
         "ecs:RegisterTaskDefinition",
+				"ecs:DeregisterTaskDefinition",
```

Comment:
Awesome PR. I couldn’t help but notice this extra indentation…

Reply:
I have no idea why my editor decided to indent that with tabs, I think it is trolling me